### ISSUE TITLE
Remove Ruby dependency

### ISSUE THREAD

**Issue by scottkellum** [at-import/sass-a11y#3]
Currently this has a strict Ruby dependency and is tightly coupled with `sassy-math`. This should be refactored so that it is pure Sass, and ideally with LibSass support.

**Comment by Snugug** [comment]
Yes, it does, because it has a hard requirement on irrational exponents. Once Sass supports exponents, we can remove the Ruby dependency.

**Comment by scottkellum** [comment]
I may work on this at some point soon for a project.

**Comment by Snugug** [comment]
Ok.

**Comment by scottkellum** [comment]
saving this here for reference: http://lamarotte2.blogspot.com/2011/04/hand-calculation-of-fractional-decimal.html
